modules/nw-operator-cr.adoc

```diff
@@ -142,13 +125,14 @@ defaultNetwork:
 <2> Specify only if you want to override part of the OpenShift SDN
 configuration.
 
-<3> Configures the network isolation mode for `OpenShiftSDN`. The allowed values
+<3> Configures the network isolation mode for OpenShift SDN. The allowed values
```

Comment:
Same as previous comment. I only see the first sentence appearing in the preview build.